File: fizzbuzz/java/alyssanewhart/FizzBuzz.java
public class FizzBuzz {

    public static String FizzBuzz(int n) {
        String result = "";
        for (int i = 1; i <= n; i++) {
            result = "";
            if (i % 3 == 0)
                result += "Fizz";
            if (i % 5 == 0)
                result += "Buzz";
            if (i % 7 == 0)
                result += "Ping";
            if (i % 11 == 0)
                result += "Pong";

            if (result.length() == 0)
                result += Integer.toString(i);

            System.out.println(result);
        }
        return result;
    }
    public static void main(String[] args) {

        int n = Integer.parseInt(args[0]);
        FizzBuzz(n);
    }
}

File: fizzbuzz/python/andrew/fizzbuzz.py
import sys

def fizzbuzz(num):
    result = []

    if (num < 1):
        return "Error input must be greater than zero."

    for n in range(1, num + 1):
        temp = ""
        if (n % 3) == 0:
            temp += "Fizz"
        if (n % 5) == 0:
            temp += "Buzz"
        if (n % 7) == 0:
            temp += "Ping"
        if (n % 11) == 0:
            temp += "Pong"
        if temp == "":
            temp += str(n)
        result.append(temp)
    return result

if __name__ == "__main__":
    num = int(sys.argv[1])
    if (num > 0):
        for n in fizzbuzz(num):
            print(n)
    else:
        print("Error input must be greater than zero.")
